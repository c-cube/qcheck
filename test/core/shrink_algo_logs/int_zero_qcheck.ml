open QCheck

let ints_are_0 =
  Test.make ~name:"ints are 0"
    int (Log.shrinks_int @@ fun i -> i = 0)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand ints_are_0 with Test.Test_fail _ -> ()
