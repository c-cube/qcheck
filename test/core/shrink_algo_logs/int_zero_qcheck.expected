fails -3285013039971199785
holds 0
fails -1642506519985599893
holds 0
fails -821253259992799947
holds 0
fails -410626629996399974
holds 0
fails -205313314998199987
holds 0
fails -102656657499099994
holds 0
fails -51328328749549997
holds 0
fails -25664164374774999
holds 0
fails -12832082187387500
holds 0
fails -6416041093693750
holds 0
fails -3208020546846875
holds 0
fails -1604010273423438
holds 0
fails -802005136711719
holds 0
fails -401002568355860
holds 0
fails -200501284177930
holds 0
fails -100250642088965
holds 0
fails -50125321044483
holds 0
fails -25062660522242
holds 0
fails -12531330261121
holds 0
fails -6265665130561
holds 0
fails -3132832565281
holds 0
fails -1566416282641
holds 0
fails -783208141321
holds 0
fails -391604070661
holds 0
fails -195802035331
holds 0
fails -97901017666
holds 0
fails -48950508833
holds 0
fails -24475254417
holds 0
fails -12237627209
holds 0
fails -6118813605
holds 0
fails -3059406803
holds 0
fails -1529703402
holds 0
fails -764851701
holds 0
fails -382425851
holds 0
fails -191212926
holds 0
fails -95606463
holds 0
fails -47803232
holds 0
fails -23901616
holds 0
fails -11950808
holds 0
fails -5975404
holds 0
fails -2987702
holds 0
fails -1493851
holds 0
fails -746926
holds 0
fails -373463
holds 0
fails -186732
holds 0
fails -93366
holds 0
fails -46683
holds 0
fails -23342
holds 0
fails -11671
holds 0
fails -5836
holds 0
fails -2918
holds 0
fails -1459
holds 0
fails -730
holds 0
fails -365
holds 0
fails -183
holds 0
fails -92
holds 0
fails -46
holds 0
fails -23
holds 0
fails -12
holds 0
fails -6
holds 0
fails -3
holds 0
fails -2
fails -1
holds 0
